sdk/src/contexts/abstracts/relayer.ts

```diff
@@ -31,4 +31,17 @@ export abstract class RelayerAbstract<
     destChain: ChainName | ChainId,
     tokenId: TokenId,
   ): Promise<BigNumber>;
+
+  parseRelayerPayload(transferPayload: Buffer): ParsedRelayerPayload {
+    return {
+      relayerPayloadId: transferPayload.readUint8(0),
+      relayerFee: BigNumber.from(
+        '0x' + transferPayload.subarray(1, 33).toString('hex'),
+      ),
+      toNativeTokenAmount: BigNumber.from(
+        '0x' + transferPayload.subarray(33, 65).toString('hex'),
+      ),
+      to: '0x' + transferPayload.subarray(65, 98).toString('hex'),
+    };
+  }
```

Comment:
The relayer payload used (for now) for Sei uses a different structure (it's a json instead of a byte array). I wanted to keep the same `ParsedRelayerPayload` interface for all chains, so I opted to make this the default implementation for the parsing method, which is the one shared by EVMs and Sui, and override with the specifics for Sei.